### DISCUSSION TITLE
[Projects Beta] Automatically add issues/PRs from other repos that I do not own but contribute to

### DISCUSSION THREAD

**Discussion by fluteds** [community/community#11547] [Projects and Issues]
As a contributor it would be super neat if we could automatically add issues to projects from repos that we do not own but contribute to. 

At the moment you have to manually copy and paste in issues/PRs URLs you've contributed to or are interested in to keep track of them and it takes a tonne of time to keep track of. Whilst I know you can see PR status and Issues in [their respective tabs](https://github.com/pulls?q=is%3Apr+is%3Aopen+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc), I prefer to use the project format to see everything on one page in an easy to read format. I can then sort down and filter based on repos and status based on what I'm working on or want to check without having 50 tabs open.

Hopefully others feel the same way about streamlining this.

**Comment by aisgbnok** [comment]
I just started organizing my contributions in a project [Projects Beta] and this automation would be very helpful! Great idea 👍.
